README.md

```diff
@@ -14,8 +14,8 @@
 <a href="#contributors-"><img alt="All Contributors" src="https://img.shields.io/badge/all_contributors-33-orange.svg"/></a>
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
 
-**Jellyseerr** is a free and open source software application for managing requests for your media library.
-It is a a fork of [Overseerr](https://github.com/sct/overseerr) built to bring support for [Jellyfin](https://github.com/jellyfin/jellyfin) & [Emby](https://github.com/MediaBrowser/Emby) media servers!
+**Jellyseerr** is a free and open-source software application for managing requests for your media library.
```

Comment:
Not a typo.

Reply:
It's an overall, but what do you want separate commit for every fix ?

Reply:
Oh wait this was accidentally